scripts/install_build_dependencies.sh

```diff
@@ -12,32 +12,28 @@ else
   DEBIAN_ARCH='amd64'
 fi
 
-apt-get update
+apt update
```

Comment:
it is not recommended to use `apt` in scripts. this is the tool for manual execution. for scripts it is better to use `apt-get`, as it is specially designed to be more stable and less colorful

Reply:
As we don't depend on the output, I hope to use `apt` as it's familiar to users. Well, I don't have a strong option about it. I'll update.

Reply:
Done.

Reply:
The same in the Dockerfile (line 78, 79). Sorry, missed that earlier(

Reply:
Done.